src/extension.ts

```diff
@@ -508,6 +516,39 @@ class Device extends vscode.TreeItem {
             [this.shellPort.toString()]);
         term.show();
     }
+    
+    captureScreenshot(): void {

```

Comment:
I suppose we _could_... why? We don't wait for completion regardless because this captures the screenshot and invokes a user action without returning any data.